packages/web/src/ui/views/viewer.js

```diff
@@ -26,6 +26,7 @@ let render
 let viewerOptions
 let camera = perspectiveCamera.defaults
 let controls = orbitControls.defaults
+controls.drag = 1;
```

Comment:
first thought is jscad uses standard.js style rules, which specifies no EOL semi-colons, spaces between "}else{", etc.